From 393a5ba09f7c5360d2e5066c23d43bb82d856173 Mon Sep 17 00:00:00 2001
From: David Herrmann <dh.herrmann@gmail.com>
Date: Thu, 11 Sep 2014 15:30:56 +0200
Subject: [PATCH] terminal: drop redundant assertion

This assertion is already there two lines down. Drop the redundant
assertion.
---
 src/libsystemd-terminal/idev-evdev.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/libsystemd-terminal/idev-evdev.c b/src/libsystemd-terminal/idev-evdev.c
index 241743c3fd..6509d1011e 100644
--- a/src/libsystemd-terminal/idev-evdev.c
+++ b/src/libsystemd-terminal/idev-evdev.c
@@ -858,7 +858,6 @@ static int managed_evdev_new(idev_element **out, idev_session *s, struct udev_de
         int r;
 
         assert_return(s, -EINVAL);
-        assert_return(s->context->sysbus, -EINVAL);
         assert_return(s->managed, -EINVAL);
         assert_return(s->context->sysbus, -EINVAL);
         assert_return(ud, -EINVAL);
